bench/workgen/workgen.cpp

```diff
@@ -1902,10 +1909,25 @@ ThreadRunner::op_run(Operation *op)
 
         do {
             // Wait for transactions to complete before stopping.
+            bool has_mirror;
             for (int count = 0; (!_stop || _in_transaction) && count < op->_repeatgroup; count++) {
                 for (std::vector<Operation>::iterator i = op->_group->begin();
                      i != op->_group->end(); i++) {
-                    WT_ERR(op_run_setup(&*i));
+                    auto [table_uri, tint] = op_get_table(&*i);
+                    {
+                        const std::shared_lock lock(*_icontext->_dyn_mutex);
```

Comment:
We should refactor this into a utility function

Reply:
Good idea! Created one in 37101aacc478ba47c50298c9168c3b39c70746e5.